src/BackupMetadata.cs

```diff
@@ -5,31 +5,45 @@ namespace AgDatabaseMove
   using SmoFacade;
 
 
-  /// <summary>
-  ///   Occasionally we wind up with the same entry for a backup on multiple instance's msdb.
-  ///   For now we'll consider these backups to be equal despite their file location,
-  ///   but perhaps there's value in being able to look for the file in multiple locations.
-  /// </summary>
-  public class BackupMetadataEqualityComparer : IEqualityComparer<BackupMetadata>
+  public class BackupEqualityComparer : IEqualityComparer<BackupMetadata>

```

Comment:
Needed to borrow this from the other PR to fix the issue here:
https://github.com/factset/AgDatabaseMove/pull/101/files#diff-0b43e3447245b624777b212abcd1a5a74ed28b239eb4797e66e7ce06b3288e62R8